code/game/verbs/ooc.dm

```diff
@@ -105,15 +105,28 @@
 	log_ooc("(LOCAL) [mob.name]/[key] : [msg]",ckey=key_name(mob))
 
 	var/mob/source = src.mob
+	var/mob/eyesource
```

Comment:
this var is technically unnecessary.

You can just write the entire sequence after the first for-loop.